packages/opentoonz/opentoonz.pacscript

```diff
@@ -0,0 +1,55 @@
+name="opentoonz"
+version="1.5.0"
+url="https://github.com/opentoonz/opentoonz/archive/refs/tags/v$version.tar.gz"
+build_depends="libboost-all-dev qtbase5-dev libqt5svg5-dev qtscript5-dev qttools5-dev libqt5opengl5-dev qtmultimedia5-dev libqt5serialport5-dev libsuperlu-dev liblz4-dev libusb-1.0-0-dev liblzo2-dev libpng-dev libjpeg-dev libglew-dev freeglut3-dev libfreetype6-dev libjson-c-dev qtwayland5 libmypaint-dev libopencv-dev libturbojpeg-dev libmypaint-dev libopenblas-dev libegl1-mesa-dev libgles2-mesa-dev libglib2.0-dev liblzma-dev"
+depends="qtchooser qt5-qmake qtbase5-dev-tools qttools5-dev-tools libqt5multimedia5-plugins qtwayland5 libgsl25 build-essential git cmake pkg-config"
+description="An open-source full-featured 2D animation creation software"
+hash="fba0f381fb71ffda89e6423b4d461450d4851f93830342e69e36313070af5990"
+maintainer="echometerain <echometer@disroot.org>"
+gives="opentoonz"
+breaks="${name}-git, ${name}-deb, ${name}-app, ${name}-deb, ${name}-bin"
```

Comment:
@D-Brox and @NarutoXY ahh... so handle this. If this is a real problem then all the scripts have broken `breaks`

Reply:
The `breaks` fix has already been merged to `develop`, but we still haven't released 1.6.2, so this pacscript one may still not work on the CI